inbc-program/README.md

```diff
@@ -178,20 +178,56 @@ INBC is only supporting the application update portion of AOTA.
 
 ### Usage
 ```
-inbc aota {--uri, -u=URI} 
-   [--app, -a APP_TYPE; default="application"] 
-   [--command, -c COMMAND; default="update"]
+inbc aota {--app, -a APP_TYPE} {--command, -c COMMAND}
+   [--uri, -u URI]
+   [--version, -v VERSION]
+   [--containertag, -ct CONTAINERTAG]
+   [--file, -f FILE]
    [--reboot, -rb REBOOT; default="no"]
-   [--username, -un USERNAME] 
+   [--username, -un USERNAME]
+   [--dockerusername, -du DOCKERUSERNAME]
+   [--dockerregistry, -dr DOCKERREGISTRY]
```

Comment:
required params should be in {} and optional in [].

Reply:
@nmgaston Yes only command and application is required as they are the common/required ones between docker-application/docker-compose and the rest are optional depending on what is chosen as the command/application